status_gateways.php Bootstrap conversion

sbeaver-netgate opened the pull request.

sbeaver-netgate committed:
status_gateways.php Bootstrap conversion

sbeaver-netgate committed:
Corrected background color

sbeaver-netgate committed:
status_gateway_groups c- conversion complete

Eliminated outer table
Changed inner table parameters
Eliminated echo & print statements
Adjusted indenting and formatting

SjonHortensius commented on usr/local/www/status_gateways.php:
```diff
@@ -53,109 +53,116 @@
+						<td>
+							<table>
+								<tr>
+									<td bgcolor="<?=$bgcolor?>">&nbsp;<?=$online?>
```
Why does this TD need to be nested in 3 tables? I propose you add the lastchange in the same cell. Also, it would be great if we could convert the background-color to an icon; did you have a look at the set for appropriate icons?

I'd rewrite this to simply `<td>$online<i> last checked $lastcheck</i></td>` instead of `<td><table><tr><td><table><tr><td>`

sbeaver-netgate replied:
Thanks. I’ll look at that.

> On Apr 22, 2015, at 8:05 AM, SjonHortensius notifications@github.com wrote:
> 
> In usr/local/www/status_gateways.php https://github.com/SjonHortensius/pfsense/pull/105#discussion_r28865706:
> 
> > -                        ?>
> > -           </table>
> > -                  </td>
> > -         <td class="listbg"> <?=$gateway['descr']; ?></td>
> > -                </tr>
> > -       <?php } ?>
> > -              </table>
> > -      </div>
> >   -</td></tr>
> >   -</table>
> >   +?>
> > -                   <tr>
> > -                       <td>
> > -                           <table>
> > -                               <tr>
> > -                                   <td bgcolor="<?=$bgcolor?>">&nbsp;<?=$online?>
> >   Why does this TD need to be nested in 3 tables? I propose you add the lastchange in the same cell. Also, it would be great if we could convert the background-color to an icon; did you have a look at the set for appropriate icons?
> 
> I'd rewrite this to simply <td>$online<i> last checked $lastcheck</i></td> instead of <td><table><tr><td><table><tr><td>
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/105/files#r28865706.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com mailto:sbeaver@netgate.com | www.netgate.com http://www.netgate.com/

SjonHortensius commented on usr/local/www/status_gateways.php:
```diff
@@ -53,109 +53,116 @@
+<?php	foreach ($a_gateways as $gname => $gateway) {
+?>
+		<tr>
+			<td class="listlr">
```
Please; remove `class="listlr"`. `id="mainarea"` is also useless and should be removed

sbeaver-netgate committed:
status_gateways.php conversion complete

Two tables eliminated
Colors defined to eliminate magic
Bootstrap column width specified
All obsolescent classes and elements removed
